src/napari_time_slicer/__init__.py

```diff
@@ -12,6 +12,7 @@
 import numpy as np
 # most imports here are just for backwards compatbility
 from ._workflow import WorkflowManager, CURRENT_TIME_FRAME_DATA, _get_layer_from_data, _break_down_4d_to_2d_kwargs, _viewer_has_layer
+from .TimelapseConverter import TimelapseConverter
```

Comment:
Can we rename this file to fulfill python conventions? (lower case and underscores instead of CamelCase)